packages/mdc-top-app-bar/fixed/foundation.ts

```diff
@@ -21,39 +21,33 @@
  * THE SOFTWARE.
  */
 
+import {MDCTopAppBarAdapter} from '../adapter';
 import {cssClasses} from '../constants';
-import MDCTopAppBarAdapter from '../adapter';
-import MDCTopAppBarFoundation from '../foundation';
+import {MDCTopAppBarFoundation} from '../standard/foundation';
 
-/**
- * @extends {MDCTopAppBarFoundation<!MDCFixedTopAppBarFoundation>}
- * @final
- */
 class MDCFixedTopAppBarFoundation extends MDCTopAppBarFoundation {
   /**
-   * @param {!MDCTopAppBarAdapter} adapter
+   * State variable for the previous scroll iteration top app bar state
    */
-  constructor(adapter) {
+  private wasScrolled_ = false;
+
+  /* istanbul ignore next */
```

Comment:
Is this something you also wanted to cover in the istanbul PR?

Reply:
Which PR are you referring to?

I added a descriptive comment to explain why this is needed.